app/views/shared/_alerts.html.erb

```diff
@@ -1,6 +1,6 @@
-<% flash.to_hash.slice('alert', 'info', 'notice', 'error').each do |name, msg| %>
+<% flash.to_hash.slice('notice', 'success', 'info', 'alert', 'error').each do |name, msg| %>
   <div class="global-alert global-alert--<%= name %>">
-    <div class="global-alert__content-container">
+    <div class="global-alert__content-container global-alert__content-container--<%= name %>">
       <strong class="global-alert__heading"><%= name.titleize %></strong>
       <%= content_tag :p, msg.html_safe, class: 'global-alert__message t-flash-message' %>
```

Comment:
I don't know. I would have thought that all content in here would be internal to the app, but maybe not.